### PR TITLE
Prepare publication

### PR DESCRIPTION
**Pull request by timtroendle**
* Remove biofuels data, following https://github.com/calliope-project/euro-calliope/issues/194.
* Update license formulations.
* Remove overarching license in favour of dataset-specific licenses.

### CODE REVIEW

**Comment by timtroendle** [comment]
I intend to publish the datasets in exactly the way they are structured within this repo.

**Review by brynpickering — APPROVED** [review]
This works by me.

Just one point: everything points to the final CSV only including "relevant" countries. Yet, I see all countries in the world in there! Shouldn't it just be european countries?

**Comment by timtroendle** [comment]
That's correct and that's why I removed the word "Europe" from the file name. :D 

We could filter here, but then we are replicating the list of countries from Euro-Calliope. The overhead is small anyway, that's why I felt I'd keep it this way.

File: hydro-generation/preprocess-data.ipynb [hunk 1]
```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Preprocess IRENA hydro generation data\n",
     "\n",
-    "This notebook preprocesses IRENA data from their \"Renewable energy statistics\" publication. It reduces the dataset to generation of hydro power in the relevant countries. This notebook creates the `./data/irena/hydro-generation-europe.csv` dataset. The original IRENA dataset is not part of this repository and must be retrieved manually to run this notebook."
+    "This notebook preprocesses IRENA data from their \"Renewable energy statistics\" publication. It reduces the dataset to generation of hydro power in the relevant countries. This notebook creates the `./hydro-generation/hydro-generation.csv` dataset. The original IRENA dataset is not part of this repository and must be retrieved manually to run this notebook."
```
Comment (brynpickering) [inline hydro-generation/preprocess-data.ipynb]: Update to remove 'It reduces the dataset to generation of hydro power in the relevant countries' to match the removal of `-europe`

**Comment by brynpickering** [comment]
Leaving it this way is fine by me. Just need to update the README as per my comment.

**Comment by timtroendle** [comment]
Done.

**Pull request merged**